### DISCUSSION TITLE
Word+Zotero CSL style to Pandoc to Latex question

### DISCUSSION THREAD

**Discussion by ioanmuntean** [retorquere/zotero-better-bibtex#2836] [General]
I am using the Better Bibtex when I import files from Word+Zotero to Latex through Pandoc. This worked like a charm for about 3 years and my citations in Word had this format: [@battermanAutonomyTheoriesExplanatory2017]. I wrote a number of LUA filters and Pandoc was converting these to entries like \cite[battermanAutonomyTheoriesExplanatory2017]. I also managed to do some work for prefixed and suffixed citations in Zotero.
Here is what happens now. When I inspect the CSL style called Beete Bibtex Citekeys I get this: [@Citation Key: primieroInformationPhilosophyComputer2016c]
There is an extra "Citation Key" text before the @ sign. Let me know if this is an expected behavior.
I can modify my LUA filter in Pandoc to ignore the "citation key" text but is there a rationale for this change? Thanks in advance!

**Comment by retorquere** [comment]
Pandoc can do this natively these days: https://retorque.re/zotero-better-bibtex/citing/migrating/index.html

**Reply by retorquere** [reply]
This one slipped under the radar. apologies for the long delay
